### PR TITLE
add google workspace commands

### PR DESCRIPTION
**Pull request by hirosassa**
## What

This PR adds Google Workspace products' jumper commands.
I implemented commands for following products:

- drive
- document
- spreadsheets
- slides
- forms

relates #106 

## Why

It will be useful.

### CODE REVIEW

**Comment by hirosassa** [comment]
@KeisukeYamashita plz review!

**Comment by KeisukeYamashita** [comment]
Thanks, I will take a look tonight(JST)🙏

**Comment by KeisukeYamashita** [comment]
Sorry, I will take look tomorrow 🙏 I can't make it today🙇🏻‍♂️

File: README.md [hunk 1]
```diff
@@ -382,6 +383,29 @@ $ biko g s -q "How to configure biko"
...
+$ biko googleworkspace [product] [flag(s)]
+# or
+$ biko gw [product] [flag(s)]
+```
+
+| Product      | What                       | Command                      | Flags(Optional) |
+| :----:       | :----:                     | :----:                       | :----:        |
+| drive        | Search on Google Drive     | `drive`, `dr`                | `--query, -q` |
+| document     | Search on Google Docs      | `document`, `dc`             | `--query, -q` |
+| document     | Create a new Google Docs   | `document`, `dc`             | `--new, -n`   |
```
Comment (KeisukeYamashita) [inline README.md:400]: I would like to use `doc`  🙏 

```suggestion
| document     | Search on Google Docs      | `document`, `doc`             | `--query, -q` |
| document     | Create a new Google Docs   | `document`, `doc`             | `--new, -n`   |
```

File: cli/googleworkspace.go [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+			return browser.Open(c, g)
+		},
+	}
+
+}
+
+func newDocumentCmd() cli.Command {
+	return cli.Command{
+		Name:    "document",
+		Aliases: []string{"dc"},
```
Comment (KeisukeYamashita) [inline cli/googleworkspace.go:72]: I think `doc` is understandable than `dc` so I want to make this change 🙏 

```suggestion
		Aliases: []string{"doc"},
```

**Review by KeisukeYamashita — CHANGES_REQUESTED** [review]
Thank you for raising this PR 🥇 
Tested and working well, I have one request 🙏

File: README.md (hunk 1 above)
Comment (hirosassa) [reply README.md]: Thanks for your comment! fixed in e42cd96

File: cli/googleworkspace.go (hunk 2 above)
Comment (hirosassa) [reply cli/googleworkspace.go]: fixed!

**Review by KeisukeYamashita — APPROVED** [review]
Thank you very much !!

**Comment by KeisukeYamashita** [comment]
Released in [v0.0.20](https://github.com/KeisukeYamashita/biko/releases/tag/v0.0.20), thank you as always 🥇

**Pull request merged**